app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/onboarding/DeterminingIfOrganisationIsTrustController.scala

```diff
@@ -88,29 +90,32 @@ class DeterminingIfOrganisationIsTrustController @Inject()(
           .bindFromRequest()
           .fold(
             formWithError =>
-              BadRequest(doYouWantToReportForATrustPage(formWithError, controllers.routes.StartController.weNeedMoreDetails())), {
-              isReportingForTrust =>
-                updateSession(sessionStore, request)(
-                  _.copy(
-                    journeyStatus = Some(
-                      DeterminingIfOrganisationIsTrust(
-                        determiningIfOrganisationIsTrust.ggCredId,
-                        Some(isReportingForTrust),
-                        None
-                      )
+              BadRequest(
+                doYouWantToReportForATrustPage(formWithError, controllers.routes.StartController.weNeedMoreDetails())
```

Comment:
looks like you hit return here by accident

Reply:
I think that was scalafmt making the long line shorter